flow/raster_cache.cc

```diff
@@ -27,6 +28,12 @@ RasterCacheResult::RasterCacheResult(sk_sp<SkImage> image,
                                      const SkRect& logical_rect)
     : image_(std::move(image)), logical_rect_(logical_rect) {}
 
+void RasterCacheResult::drawSnapshot(SkCanvas* canvas) const {
+  SkAutoCanvasRestore auto_restore(canvas, true);
+  canvas->resetMatrix();
+  canvas->drawImage(image_, 0, 0, nullptr);
```

Comment:
My intention here is to render them in Src blend mode because that is the definition of the operation. The image should have been calculated to include the desired alpha for every pixel when it was created.

I am assuming that a null paint means a Src mode copy, but I notice that drawImage doesn't define its behavior when paint is null - it admits that it applies paint if paint is non-null, but doesn't document what it does if paint is null. I'll need to investigate that further and compare to what the saveLayer with a backdrop does (via SkCanvas::DrawDeviceWithFilter).